src/App.tsx

```diff
@@ -1,42 +1,36 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
+import { Tab } from './components/Tabs/types/interface';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [selectedTabId, setTabId] = useState(tabs[0].id);
+  const [title, setTitle] = useState('Tab 1');
+
+  const onTabSelected = (currentTab: Tab) => {
```

Comment:
```suggestion
  const handleTabSelected = (currentTab: Tab) => {
```

Reply:
I can't change the name of the function, because our test needs a current name of the function. Also, our task requirement asks for that name.
![image](https://user-images.githubusercontent.com/106397016/234869693-14dc0042-16e9-4ef2-811a-2b01a3398755.png)

Reply:
No, just name in props